cmd/listsuites/listsuites.c

```diff
@@ -50,13 +50,12 @@ main(int argc, char **argv)
         }
         fprintf(stdout,
                 "%s:\n" /* up to 37 spaces  */
-                "  0x%04hx %-5s %-5s %-8s %3hd %-6s %-8s %-4s %-8s %-11s\n",
+                "  0x%04hx %-5s %-5s %-8s %3hd %-6s %-8s %-4s %-11s\n",
```

Comment:
Is there any chance that someone is parsing this output?  I think that you should add a fixed "Disabled" string here.